go/dism/dism.go

```diff
@@ -37,6 +38,8 @@ const (
 	DISM_MOUNT_READONLY        = 0x00000001
 	DISM_MOUNT_OPTIMIZE        = 0x00000002
 	DISM_MOUNT_CHECK_INTEGRITY = 0x00000004
+
+	DISMAPI_S_RELOAD_IMAGE_SESSION_REQUIRED syscall.Errno = 0x00000001
```

Comment:
zdism.go already has an error wrapper (errnoErr). It might be more consistent to track and translate this error there, along with the others. I'm a bit wary of whether 0x00000001 will be universally the same error for this API, but the documentation isn't terribly helpful on this point.

Reply:
I saw that but didn't want to manually modify an auto-generated file. But seems like we can't influence the generation to include additional error codes, right?